### DISCUSSION TITLE
Migrate to new server

### DISCUSSION THREAD

**Discussion by farokh-ny** [postfixadmin/postfixadmin#876] [Q&A]
I'm currently running postfixadmin 3.3.11 on a vm under esxi 7.0u2 running ubuntu 22.04.5.

Long story short, the vm appears to have an issue where the vm is running on snapshot disks even though there are no snapshots.
I've been trying to fix it for a bit, but I think it's just easier if I create a new vm and move everything necessary over, however I can't seem to find any docs or web pages on exactly how to do that.

Any pointers on how to accomplish this?

I'm guessing that I need to move the sql database, the /var/vmail folder, the /etc/postfix folder and the /etc/dovecot folder, but I'm not sure what else.

Thanks.

**Comment by DavidGoodwin** [comment]
Hi @farokh-ny - 

This depends a lot on your platform's configuration, how the software has been installed.

Some questions to ask are probably : 

 * do you have a separate database server?
 * was postfixadmin installed through a .deb or from a .tar.gz or a git clone of this repository?
 * Do you need dovecot (for password hashing) to be installed on the web server?
 * What other services are running on the server? (dovecot? dkim? greylisting daemon?)
 * What version of PHP ...
 * What webserver and what configuration for it?
 * Does your site/server need an ssl cert? if so, where's that? how is it being renewed?

There's a reasonable chance an /etc/postfixadmin directory exists, in which case copy that too.

Good luck!

**Reply by farokh-ny** [reply]
> Hi @farokh-ny -
> 
> This depends a lot on your platform's configuration, how the software has been installed.
> 
> Some questions to ask are probably :
> 
> * do you have a separate database server?

No, mysql is on this same server

> * was postfixadmin installed through a .deb or from a .tar.gz or a git clone of this repository?
.tar.gz
> * Do you need dovecot (for password hashing) to be installed on the web server?

yes, it's used for imap etc.

> * What other services are running on the server? (dovecot? dkim? greylisting daemon?)

dovecot, dkim, fail2ban

> * What version of PHP ...

8.1.2

> * What webserver and what configuration for it?

apache 2.4.52 (used for postfixadmin, roundcube)

> * Does your site/server need an ssl cert? if so, where's that? how is it being renewed?

let's encrypt

> 
> There's a reasonable chance an /etc/postfixadmin directory exists, in which case copy that too.

it does, thanks.

> 
> Good luck!

Thanks!
